### ISSUE TITLE
errorformat for compiler feature does not work

### ISSUE THREAD

**Issue by bagohart** [blankname/vim-fish#7]
Hi,
nice to see this plugin get some love again!
I'm currently trying to develop some fish scripts, and I noticed that the error format doesn't fit with the actual error messages.
The error format is
`  errorformat=%Afish: %m,%-G%*\ ^,%-Z%f (line %l):%s`
but I get this error message:
```
integrator.fish (line 24): Missing end to balance this if statement
if nope
^
warning: Error while reading file integrator.fish
```
which just fills the quickfix list with 5 lines.

If I understand this correctly, this problem has been known since 2013 by the original author of this plugin:
https://github.com/fish-shell/fish-shell/issues/539
and embarassingly, the feature still doesn't exist on the fish side of things.

So if I understand this correctly, there's currently nothing we can do in this plugin, so it would be good to add a note to the documentation, so the next guy doesn't have to waste half an hour to track this down, like I just did :)
(Unless you have in the meantime solved this riddle and there's just a little setting I'm missing, in which case I'd be grateful if you could share that secret...)

**Comment by blankname** [comment]
@bagohart Please try #8 and let me know if it works for you.

**Comment by blankname** [comment]
Closed by #8.
